src/main/java/pl/lodz/p/edu/flanki/services/EventService.java

```diff
@@ -43,4 +48,15 @@ public void createEvent(final EventDto eventDto) {
                 .map(eventMapper::toDto)
                 .collect(Collectors.toList());
     }
+
+    public void joinEvent(JoinEventDto joinEventDto) {
+        User user = userService.getUser();
+        Event event = eventRepository.findById(UUID.fromString(joinEventDto.getUuid()).toString()).orElseThrow(
```

Comment:
Instead of using joinEventDto.getUuid() we could simply change String type to UUID type in joinEventDto (then client who is calling this endpoint will get error, that he didn't put proper id format)